src/passkey_child/passkey_child_common.c

```diff
@@ -96,6 +100,8 @@ parse_arguments(int argc, const char *argv[], struct passkey_data *data)
          _("Require user-verification"), "true|false"},
         {"debug-libfido2", 0, POPT_ARG_NONE, NULL, 'd',
          _("Enable debug in libfido2 library"), NULL},
+        {"write-pem", 0, POPT_ARG_NONE, NULL, 'p',
```

Comment:
I wonder if `write-pem` is still needed when there is the formatting option? If you prefer to keep it I would suggest to make it a `POPT_ARG_STRING` option and require a file name where to write the PEM formatted key into.

Reply:
In the end I removed it